submissions/2020-10-06-OpenFF-Phenyl-Set/compute.json

```diff
@@ -0,0 +1,87 @@
+{
+  "qc_specifications": {
+    "openff-1.0.0": {
+      "method": "openff-1.0.0",
```

Comment:
I think for openff spec we now want to use `openff_unconstrained-1.0.0` even though the constraints are automatically removed from any forcefield that goes through the new openmm harness this just makes it clear to the user of the data, or maybe we could put in the spec description that it is unconstrained? @trevorgokey?